### PR TITLE
adding ops.finfo

### PR DESCRIPTION
**Pull request by anas-rz**
This PR adds a multi-backend way to return machine limits for different types.

I couldn't figure out any other way to do this in multibackend manner.

CC: @fchollet.

### CODE REVIEW

**Comment by anas-rz** [comment]
Do you think this one is better? Returning `numpy` type from the `torch` doesn't seem feasible.  the min and max attributes work the same across all backends.

File: keras/backend/jax/core.py [hunk 1]
```diff
@@ -374,3 +373,11 @@ def device_scope(device_name):
     else:
         jax_device = device_name
     return jax.default_device(jax_device)
+
+
+def finfo(dtype):
+    return jnp.finfo(dtype)
```
Comment (fchollet) [inline keras/backend/jax/core.py:379]: Since the values should be the same regardless of the backend, we don't really need to access the per-backend `finfo`, do we?

Comment (anas-rz) [reply keras/backend/jax/core.py:379]: You are right! I thought this op is useful in some attention-related ops like masking, etc. I tried testing them and they show the same values. We could simply make use of np.finfo.

Closing this PR.
